libs/payments/ui/src/lib/client/components/CheckoutForm/index.tsx

```diff
@@ -251,15 +253,17 @@ export function CheckoutForm({
                 className="mt-6"
               />
             ) : (
-              <PrimaryButton
+              <SubmitButton
```

Comment:
In this case is `SubmitButton` working as intended? I didn't see a spinner on submit.